e2e/tests/room/02-room.spec.js

```diff
@@ -354,6 +361,26 @@ describe('Room screen', () => {
 			});
 		});
 
+		describe('Mark message as unread', async() => {
+			it('should mark message as unread', async() => {
+				await tapBack();
+				await navigateToUser(data.users.alternate.username);
+				await mockMessage('message')
+				await expect(element(by.label(`${ data.random }message`)).atIndex(0)).toExist();
+				await tapBack();
+				await logout();
+				await navigateToLogin();
+				await login(data.users.alternate.username, data.users.alternate.password);
+				await navigateToUser(data.users.regular.username);
+				await element(by.label(`${ data.random }message`)).atIndex(0).longPress();
+				await expect(element(by.id('action-sheet'))).toExist();
+				await expect(element(by.id('action-sheet-handle'))).toBeVisible();
+				await element(by.id('action-sheet-handle')).swipe('up', 'fast', 0.5);
+				await element(by.label('Mark Unread')).tap();
```

Comment:
You're not really testing if the channel is unread.
It's a little harder to test it without adding a new testID, but I think we can do it using accessibilityLabel.
Something like `${ channel } unread`. We can do it afterwards though.
For now just make sure it navigated to RoomsListView and the room is there.